docs/fsharp/language-reference/slices.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Slices (F#)
+description: Learn about how to use slices for existing F# data types and how to define your own slices for other data types.
+ms.date: 01/22/2019
+---
+
+# Slices
+
+In F#, a slice is a subset of a data type. To be able to take a slice from a data type, some implementation of the `GetSlice` method must be implemented either in the data type or in a [type extension](type-extensions.md) that is in scope. This article will explain how to take slices from existing F# types and how to define your own.
+
+Slices are similar to [indexers](members/indexed-properties.md), but instead of yielding a single value from the underlying data structure, they yield multiple ones.
+
+## Basic slicing with F# lists and arrays
+
+The most common data types that are sliced are F# lists and arrays. The following example demonstrates how to do this with lists:
+
+```fsharp
+// Generate a list of 100 integers
+let fullList = [ 1 .. 100 ]
+
+// Create a slice from indices 1-5 (inclusive)
+let smallSlice = fullList.[1..5]
+printfn "Small slice: %A" smallSlice
+
+// Create a slice from the beginning to index 5 (inclusive)
+let unboundedBeginning = fullList.[..5]
+printfn "Unbounded beginning slice: %A" unboundedBeginning
+
+// Create a slice from an index to the end of the list
+let unboundedEnd = fullList.[94..]
+printfn "Unbounded end slice: %A" unboundedEnd
+```
+
+Slicing arrays is also identical:
+
+```fsharp
+// Generate an array of 100 integers
+let fullArray = [| 1 .. 100 |]
+
+// Create a slice from indices 1-5 (inclusive)
+let smallSlice = fullArray.[1..5]
+printfn "Small slice: %A" smallSlice
+
+// Create a slice from the beginning to index 5 (inclusive)
+let unboundedBeginning = fullArray.[..5]
+printfn "Unbounded beginning slice: %A" unboundedBeginning
+
+// Create a slice from an index to the end of the list
+let unboundedEnd = fullArray.[94..]
+printfn "Unbounded end slice: %A" unboundedEnd
+```
+
+## Slicing multidimensional arrays
+
+F# supports multidimensional arrays in the F# core library. As with one-dimensional arrays, they can be useful to slices of. However, the introduction of additional dimensions mandates a slightly different syntax so that you can take slices of specific rows and columns.
```

Comment:
"they can be useful to slices of" --> slices of multidimensional arrays can be useful.